packages/credit-protocol/index.ts

```diff
@@ -285,9 +287,9 @@ export default class CreditProtocol {
     })
   }
 
-  async createCreditRecord(ucac, address1, address2, amount, memo) {
+  async createCreditRecord(ucac, address1, address2, amount, memo, fromLink = false) {
```

Comment:
:point_up: Building upon my comment above regarding named parameters -- because `createCreditRecord()` here takes 6 parameters and `CreditRecord`'s constructor takes 7, and we just added one, it's not as clear which parameter is "missing". Only with close inspection of each use of `createCreditRecord()` and `new CreditRecord()` would we be able to identify whether each call is happening with the right number of arguments and with the right positions of those arguments.